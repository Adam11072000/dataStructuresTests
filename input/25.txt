init 5736
getHighestScoredLabel 716 -1
addImage -9
getHighestScoredLabel 3748 -10
quit