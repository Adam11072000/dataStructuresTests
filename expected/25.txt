init done.
getHighestScoredLabel: INVALID_INPUT
addImage: INVALID_INPUT
getHighestScoredLabel: INVALID_INPUT
quit done.
